tests/webgl/llt-three/index.html

```diff
@@ -61,13 +72,6 @@ <h2>msc_basis_transcoder test</h2>
     const mesh = new THREE.Mesh(geometry, material);
     scene.add(mesh);
 
-    var dict = {
```

Comment:
Thanks for noticing and removing this leftover from troubles using constants as keys. I was trying to understand if I was doing something wrong in the way I was specifying the dictionary. I wasn'.t I looked further and I discovered the bracket notation for doing what I wanted.